packages/http-caching-proxy/src/__tests__/integration/http-caching-proxy.integration.ts

```diff
@@ -41,7 +41,7 @@ describe('HttpCachingProxy', () => {
 
   it('proxies HTTP requests', async function() {
     // Increase the timeout to accomodate slow network connections
-    // tslint:disable-next-line:no-invalid-this
+    /* eslint-disable-next-line no-invalid-this */
```

Comment:
```suggestion
    // eslint-disable-next-line no-invalid-this
```

There are too many places to point out, please search for them yourself. Here is a pattern that works well for me: `/* eslint-disable-next-line`